### ISSUE TITLE
PHP-DI string function is falsely reported

### ISSUE THREAD

**Issue by shadowhand** [slevomat/coding-standard#653]
```
 28 | ERROR | [x] Type DI\string is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
```

My code in question uses [PHP-DI definition][1] for a concatenated string value:

```php
use function DI\string;

return [
    'app.cache' => string('{app.root}/var/cache'),
];
```

I assume that this is reporting incorrectly because `string` is not being recognized as a function name.

[1]: http://php-di.org/doc/php-definitions.html#string-expressions

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/6fc13b6a904e1188c8d0d993a4e87fc9e1f56152

**Comment by shadowhand** [comment]
@kukulich can you tag this into a bugfix please?
